{
  "_annotation": {
    "cve_id": "CVE-2024-41656",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-41656.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:sentry:sentry:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "26C03537-BBCF-5FF7-AA07-93425C402263",
                "versionEndExcluding": "24.7.1",
                "versionStartIncluding": "10.0.0",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}